packages/worker/src/worker.ts

```diff
@@ -924,7 +924,10 @@ export class Worker {
                     );
 
                     const { activityType } = info;
-                    const fn = this.options.activities.get(activityType);
+                    // Use the corresponding activity if it exists, otherwise, fallback to default activity function (if exists)
+                    const fn = this.options.activities.get(activityType)
```

Comment:
nit: don't do `.get()` twice for the same value.

```suggestion
                    const fn = this.options.activities.get(activityType) ??
                                   this.options.activities.get('default');
```

Reply:
done